src/borg/archiver.py

```diff
@@ -1999,17 +2128,17 @@ def process_epilog(epilog):
         `repokey` and `keyfile` use AES-CTR-256 for encryption and HMAC-SHA256 for
         authentication in an encrypt-then-MAC (EtM) construction. The chunk ID hash
         is HMAC-SHA256 as well (with a separate key).
-        These modes are compatible with borg 1.0.x.
+        These modes are compatible with Borg 1.0.x.
```

Comment:
Yeah I can unstage these.

From a writing PoV it would make sense to use `Borg` when we're talking about the project (e.g. docs, releases, community), `borg` when specifically the command is meant.